### ISSUE TITLE
main.py, no EmpowermentSkills

### ISSUE THREAD

**Issue by moonblue333** [navneet-nmk/Hierarchical-Meta-Reinforcement-Learning#1]
from maml_rl.policies.empowerment_skills import EmpowermentSkills

**Comment by navneet-nmk** [comment]
Yes. I am aware of this. I am still refactoring the code. This should be addressed in the next commit.

**Comment by fokx** [comment]
Same issue, seems the author cannot fix it for 2 years
